docs/user-guide/binned-data/filtering.ipynb

```diff
@@ -73,40 +145,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "ntemp = 100\n",
-    "sample_temperature = sc.DataArray(\n",
-    "    data=sc.array(dims=['time'], unit='K',\n",
-    "                  values=5*np.random.rand(100)+np.linspace(100, 120, num=ntemp)),\n",
-    "    coords={'time':sc.Variable(dims=['time'], unit='s',\n",
-    "                               values=np.linspace(0, end_time, num=ntemp).astype('datetime64[s]'))})\n",
-    "x = sc.linspace(dim='x', unit='m', start=0, stop=1, num=4)\n",
-    "\n",
-    "end = sc.array(dims=['x'], values=np.cumsum(sizes), unit=None)\n",
-    "begin = end.copy()\n",
-    "begin.values -= sizes\n",
-    "events = sc.DataArray(\n",
-    "    data=sc.bins(begin=begin, end=end, dim='event', data=table),\n",
-    "    coords={'x': x},\n",
-    "    attrs={'sample_temperature': sc.scalar(value=sample_temperature)})\n",
-    "for size, bucket in zip(sizes, events.values):\n",
-    "    bucket.coords['time'].values = np.linspace(0, end_time, num=size).astype('datetime64[s]')\n",
-    "    bucket.coords['tof'].values = np.concatenate(\n",
-    "        (np.concatenate(\n",
-    "            (7*width + width*np.random.randn(size//4),\n",
-    "             13*width + width*np.random.randn(size//4))),\n",
-    "         10*width + width*np.random.randn(size//2)))\n",
-    "events"
+    "da.hist().plot()"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Filtering based on existing coords\n",
-    "\n",
-    "### Extracting data based on an interval\n",
+    "### Extract time interval\n",
     "\n",
-    "We can use [scipp.bin](../../generated/functions/scipp.bin.rst) with the desired bounds to extract all data points (events) that have coord values falling within an interval:"
+    "The [mechanical strain](https://en.wikipedia.org/wiki/Strain_(mechanics)) of the steel sample in the loadframe is recorded in the metadata:"
```

Comment:
This link is broken (via github view file), I think it getting confused by the double ).
Might just be a github thing though.

Reply:
Work from a notebook.

Reply:
👍 I thought that might be the case.